### DISCUSSION TITLE
Response to "Practical IT with Jeremy Leik" review of SpiralLinux

### DISCUSSION THREAD

**Discussion by geckolinux** [SpiralLinux/SpiralLinux-project#10] [General]
_I'm responding here to [this](https://www.youtube.com/watch?v=ycUAvjp7kKg) Youtube video review since Youtube always deletes my comments. Thanks a lot to Jeremy for the comprehensive and open-minded review!_

-------------------------

Hi there Jeremy, SpiralLinux creator here. Thanks a lot for the fair and open-minded coverage! So to respond to a view moments of the video:

- 33:33 Regarding "porting" from GeckoLinux, it wasn't really possible to port anything since the base distros are totally different and the build tools (Kiwi vs. live-build) are also different. But I did try to include the same features in SpiralLinux that GeckoLinux has, although it requires different tools and ways of achieving it. And as for look and feel they are extremely similar, since like you said I am also a creature of habit and I ended up configuring the SpiralLinux spins the same as the GeckoLinux ones. ;-)
- 34:00 I'm really not sure what happened with the boot process there. But it looks like it was simply a matter of abnormal extremely slow performance. In general it seems like your Proxmox environment is performing extremely slow, at least in this video. It even took GRUB over 10 seconds just to render the menu. Maybe an issue with virtIO and/or KVM not configured? I don't have any experience with Proxmox, but I have extensively tested SpiralLinux with Gnome Boxes (on top of libvirt) and performance is acceptable, and under VirtualBox performance is excellent. Sorry I can't be of more help in diagnosing this. _(**Update:** Looks like it was just the host system overloaded by backups as you mentioned.)_
- 39:46 Yeah, LXQt is rather annoying with its "No battery" popups. The other desktop environments don't do that.
- 49:30 Yep, it's all standard Debian stuff. I made a point of not including any custom packages or SpiralLinux repositories, because I don't want the final installed system to depend on me, but rather 100% on the upstream Debian infrastructure.
- 01:00:00 Thanks for noticing the gnome-packages thing, that was pre-installed by mistake. ;-) Synaptic is far and away the best low-level GUI package manager, and it's all I recommended for working with DEB packages.
- 01:09:45 Regarding VirtualBox Guest Additions, they shouldn't be visible anywhere in the menu (I think you're remembering what you saw on the website), but they are pre-installed. What was missing from the first release was the spice-vdagent package for automatic resizing under libvirt. I'm working on the next release as we speak which will also include that, it'll be out very soon.
- 01:12:10 Sorry about the integration of printer driver searching with Gnome Software, that's kind of annoying and completely unnecessary. I should look for a way to disable that. But on the positive side, if you install it on real hardware that's not double-NATed it will actually find the network printer and auto-install it without any user intervention. It's impressively user-friendly in that sense (just using the correct Debian packages pre-installed, I can't take any credit for that.)

Again, thanks a lot for the comprehensive review and friendly comments! Let me know on the SpiralLinux forum or bug tracker if you have any other thoughts or issues with it. Cheers.

**Comment by saberking** [comment]
I was just wondering--why does Youtube delete your comments? Thanks

**Reply by geckolinux** [reply]
Just the typical artificial stupidity from Google, it lets raunchy spam through and censors legitimate commenters.
